udf/mode_last_retain_nulls.sql

```diff
@@ -0,0 +1,30 @@
+/*
+
+Returns the most frequently occuring element in an array.
+
+In the case of multiple values tied for the highest count, it returns the value
+that appears latest in the array. Nulls are retained.
```

Comment:
Can we add something along the lines of "see also: udf.mode_last which ignores null values" here, and then an equivalent callout in `udf.mode_last` for discovering this function?